### ISSUE TITLE
[Bug Report][2.5.7] List with selectable items uses aria roles incorrectly

### ISSUE THREAD

**Issue by netopyr** [vuetifyjs/vuetify#14046]
### Environment
**Vuetify Version:** 2.5.7
**Vue Version:** 2.6.14
**Browsers:** Chrome 92.0.4515.131
**OS:** Mac OS 10.15.7

### Steps to reproduce
Basically I took the first example from the documentation and ran it. https://vuetifyjs.com/en/components/list-item-groups/

### Expected Behavior
The generated code should conform to the specification of aria roles, in particular an element with the aria role *list* should have children with the role *listitem* or *group*.
https://www.w3.org/TR/wai-aria-1.1/#list

### Actual Behavior
The generated (and simplified) HTML code is shown below. The *v-list* component generates an element with the role *list*, but there are no children with the role *listitem*.
```
<div role="list">
  <div role="listbox">
    <div role="option" />
    [...]
  </div>
</div>
```

### Reproduction Link
<a href="https://codepen.io/netopyr/pen/MWmZqao" target="_blank">https://codepen.io/netopyr/pen/MWmZqao</a>

### Other comments
Omitting the outer *v-list* component fixes the issue, but I do not know what the side effects of that change would be.

**Comment by KaelWD** [comment]
#15998
